freebasic-mode.el

```diff
@@ -217,8 +217,9 @@
   ;;(start-process "./fbc" (buffer-file-name))
 					;(setq compile-cmd (concat "fbc -x ~fbout " (buffer-name) " && ./~fbout" )
    (setq compile-cmd (concat "fbc " (buffer-file-name)  " -x ./~fbout && ./~fbout && rm ./~fbout" )
```

Comment:
'setq' ends here so, ')' should be here, actually. 
Thank you for pointing this out.